privileges-grantor/README.md

```diff
@@ -0,0 +1,55 @@
+# Overview
+
+Terraform module for setting up infrastructure for [Apiary Privilege Grantor](https://github.com/ExpediaGroup/apiary-extensions/tree/master/apiary-metastore-events/apiary-metastore-consumers/privileges-grantor).
```

Comment:
Should be `[Apiary Privileges Grantor]` (plural) to match everywhere else.

Reply:
It should be. Done.